tests/bearlib/aspects/ClassTest.py

```diff
@@ -1,4 +1,4 @@
-from coalib.bearlib.aspects import Root, aspectTypeError
+from coalib.bearlib.aspects import Root, ApectTypeError
```

Comment:
... OR ... use `from coalib.bearlib.aspects import \n    (ApectTypeError as aspectTypeError)` so you do not need to modify the `pytest.raises` here and on the next file.

Reason is they will be updated anyway in https://github.com/coala/coala/pull/3894
(And after this is merged, review that PR and ensure that they update the name there.)